setup.py

```diff
@@ -0,0 +1,20 @@
+#!/bin/env python
+from setuptools import setup,find_packages
+
+setup(
+    name="pomodorotasks",
+    packages=find_packages(exclude=['test' 'test/*']),
+    install_requires=['tasklib', 'pygobject', 'dbus-python'],
+    entry_points={
+        'console_scripts': [
+            'pomodorotasks = pomodoro.client:main'
+            ],
+        'gui_scripts': [
+            'pomodoro-taskd = pomodorotasks.daemon:main',
+            ]
```

Comment:
I chose the task suffix to refer back to taskwarrior (and because the project is called pomodoroTasks2), and to avoid any confusion between any other pomodoro timers a user might have installed. 

But yes, it will automatically create a script in /usr/bin that will call the main function of `daemon.py` and `client.py` respectively. (it looks like [this](https://gist.github.com/MadeOfMagicAndWires/42d783f58f43a1581bc59fa0f9bcce51)).

I just pushed the namechange so I think we should be alright.